pandas/tests/indexes/timedeltas/test_construction.py

```diff
@@ -44,6 +44,11 @@ def test_constructor(self):
         tm.assert_index_equal(TimedeltaIndex([400, 450, 1200], unit='ms'),
                               expected)
 
+    def test_constructor_iso(self):
+        expected = timedelta_range('1s', periods=9, freq='s')
+        durations = ['P0DT0H0M{}S'.format(i) for i in range(1, 10)]
+        tm.assert_index_equal(to_timedelta(durations), expected)
```

Comment:
Minor nit but we have a pseudo-standard of doing `tm.assert_index_equal(result, expected)` so would be nice to assign to a variable called `result` in the line above and simply do the comparison on the last line